docs/sources/flow/tasks/migrate/from-otelcol.md

```diff
@@ -0,0 +1,276 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/migrate/from-otelcol/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/tasks/migrate/from-otelcol/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/tasks/migrate/from-otelcol/
+- /docs/grafana-cloud/send-data/agent/flow/tasks/migrate/from-otelcol/
+canonical: https://grafana.com/docs/agent/latest/flow/tasks/migrate/from-otelcol/
```

Comment:
```suggestion
canonical: https://grafana.com/docs/agent/latest/flow/tasks/migrate/from-otelcol/
```
New topic doesn't need aliases defined. These are only required if we move or rename a previously published topic.